.github/workflows/gendocs.yml

```diff
@@ -0,0 +1,49 @@
+name: gendocs
+
```

Comment:
This
 Code review seems to be very big with 900 changed rows. As I got it  just a small part i changed really. The other files are auto generated.
 It this is true. Create two pull requests. The fiest with rhe actual chages and the second with the auto-generated files. Even better do not keep in git auto generated files.

Reply:
The auto-generated files are the documentation and they need to stay in `nujo/docs` in order to host them at: https://vivelev.github.io/nujo/

As splitting this PR into two, I  do not how we can do that from the current state. Will see what we can do.

Reply:
Edit your branch and push the new version